authors/apps/report/test/test_report.py

```diff
@@ -0,0 +1,40 @@
+from django.apps import apps
+from django.test import TestCase
+from django.urls import reverse
+from rest_framework.test import APIClient
+import factory
+from faker import Factory
+from django.contrib.auth import get_user_model
+
+Article = apps.get_model('article', 'Article')
+faker = Factory.create()
+
+
+class UserFactory(factory.DjangoModelFactory):
+    class Meta:
+        model = get_user_model()
+
+    username = factory.Sequence(lambda n: 'map%d' % n)
+    email = factory.Sequence(lambda n: 'example_%s@map.com' % n)
+    password = factory.PostGenerationMethodCall('set_password', '1234abcd')
+
+
+class TestArticles(TestCase):
+    def setUp(self):
+        self.user = UserFactory()
```

Comment:
I have seen you have created the setup to all of the test file you created, kindly create a base class and inherit from it

Reply:
This is the only task on tests so creating a base test file will be a futile job since no other task needs to update it. See pivotal tracker stories.